docs/index.md

```diff
@@ -33,7 +33,7 @@ How-to guides provide step-by-step instructions for a wide range of tasks:
 - [Storing data in multiple application definitions](how-tos/using-multiple-appdefs.md)
 - [Representing experimental geometries](how-tos/transformations.md)
 - [Using pynxtools test framework](how-tos/using-pynxtools-test-framework.md)
-
```

Comment:
pynxtools will always create NeXus file by python, I guess what you want to stress here is
Writing NeXus files by hand i.e. without the dataconverter

I would split that section into two, a first short one where you give your experience how you have written that nexus file by hand i.e. using explicit hand written calls to e.g. h5py or other HDF5 library functions
and another main section that talks only about validation

the connection between the two is that when writing the file by hand there is no verification run thats why especially here having tools for the validation is useful, the dataconverter runs some validation  during during the convert.py calls

Reply:
yea right. Done as also Lukas suggested the same point out.